test/unit/rubygem_test.rb

```diff
@@ -215,9 +217,12 @@ class RubygemTest < ActiveSupport::TestCase
       @version_three.dependencies << create(:dependency,
         version: @version_three,
         rubygem: @dep_rubygem)
+      @version_five.dependencies << create(:dependency,
+        version: @version_five,
+        rubygem: @dep_rubygem)
     end
 
-    should "return all depended rubygems" do
+    should "return all depended rubygems (except yanked versions)" do
       gem_list = Rubygem.reverse_dependencies(@dep_rubygem.name)
```

Comment:
@olivierlacan It's not a big deal but I think "except yanked versions" doesn't need parenthesis.